plugins/modules/system/xfconf.py

```diff
@@ -32,16 +35,22 @@
     description:
```

Comment:
Please remove all ignore.txt entries in tests/sanity/ignore-2.10.txt for this module. Then the sanity tests will also tell you that you should add `type: xxx` to the documentation, and `elements: xxx` for `type: list`.

Reply:
Hi Felix, documentation is passing sanity now. I still need to write the unit tests and assert whether there is something not backwards compatible (AFAIR, there shouldn't be, but I have to double check that)

Reply:
Thanks for putting all these comments into the PR.